server/static/js/index/home_page_processing.js

```diff
@@ -0,0 +1,122 @@
+    if (onlineStatus === "Online") {
+        $row.append($("<td data-device-status class='online'>" + onlineStatus + "</td>"));
+    } else {
+        $row.append($("<td data-device-status class='offline'>" + onlineStatus + "</td>"));
+    }
+}
+
+// Callback function, called on success, for reading users data
+function drawTableUsers (data) {
+    for (let index = 0; index < data.users.length ; index++) {
+        drawRowUsers(data.users[index]);
+    }
+}
+
+// The function of drawing user table rows
+function drawRowUsers(rowDataUsers) {
+    let $row = $('<tr class="table-row table-remove-row-user">');
+    let userEnabled = rowDataUsers.active;
+
+    userEnabled = userEnabled === true ? "Yes" : "No";
+
+    $(TABLE_USERS).append($row);
+    $row.append($("<input data-user-id type='hidden'>").val(rowDataUsers.id));
+    $row.append($("<td data-user-name data-first-name data-last-name>" +
+                "" + rowDataUsers.first_name + " " + rowDataUsers.last_name + "</td>"));
+    $row.children('[data-user-name]').attr('data-first-name', rowDataUsers.first_name);
+    $row.children('[data-user-name]').attr('data-last-name', rowDataUsers.last_name);
+    $row.append($("<td data-user-email>" + rowDataUsers.email + "</td>"));
+    $row.append($("<td data-user-last-login>" + rowDataUsers.last_login_at + "</td>"));
+    
+    if (userEnabled === "Yes") {
+        $row.append($("<td data-user-enabled class='online'>" + userEnabled + "</td>"));
+    } else {
+        $row.append($("<td data-user-enabled class='offline'>" + userEnabled + "</td>"));
+    }
+
+    $row.children('[data-user-enabled]').attr('data-user-enabled', rowDataUsers.active);
+
+    $row.append($("<td><div class='user-roles-container'></div></td>"));
+
+    for (let i = 0; i < rowDataUsers.roles.length; i++) {
+        $row.children('td:last-child').children('.user-roles-container').append($("<input data-user-role type='hidden'>").val(rowDataUsers.roles[i].name));
+    }
+}
+
+// Cleaning the device table
+function removeTableDevices () {
+    $(".table-remove-row-device").remove();
+}
+
+// Clearing the user table
+function removeTableUsers () {
+    $(".table-remove-row-user").remove();
+}
+
+
+function renderTableDevices () {
+    removeTableDevices();
+    sendJSONRequest("/home/getDevicesList", null, RequestMethod.GET, beforeLoadTableDevice,
+                    drawTableDevices, debug_callback, process_failures);
+    setTimeout(function () {
+        rowSelection();
+    }, renderTableDelay);
+    afterLoadTableDevice();
+}
+
+function renderTableUsers () {
+    removeTableUsers();
+    sendJSONRequest("/home/getUsersList", null, RequestMethod.GET, beforeLoadTableUsers,
+                    drawTableUsers, debug_callback, process_failures);
+    setTimeout(function () {
+        rowSelection();
+    }, renderTableDelay);
+    afterLoadTableUsers();
+}
+
+$(document).ready(function () {
+
```

Comment:
Please, put a comment here explaining that this first call populates the table first, before going into interval loop.